packages/itwinui-react/src/core/ThemeProvider/ThemeProvider.tsx

```diff
@@ -96,19 +115,23 @@ export const ThemeProvider = React.forwardRef((props, forwardedRef) => {
     theme: themeProp = 'inherit',
     children,
     themeOptions,
+    portalContainer: portalContainerProp,
```

Comment:
i also thought about using the parent portalContainer but i'm not sure becaude it really complicates the default behavior.

<img width="1231" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/f9e8a63d-c8dc-4ccb-b8ed-6d5faccee88e">